django_tutorial/first_project/web_app/static/web_app/js/index.js

```diff
@@ -0,0 +1,385 @@
+// TODO: add browserify or webpack to build pipeline, and break this into separate js files with imports,
```

Comment:
The JS doesn't access the database directly, instead it fetches data from the API.  There are a couple `$.ajax({...})` lines in index.js that hit the API URLs.